src/main/migrations/changesets/add_iv.xml

```diff
@@ -3,7 +3,7 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
 
-    <changeSet author="chrisl" id="addColumn-iv">
+    <changeSet author="chrisl" id="addColumn-iv" logicalFilePath="src/main/migrations/changesets/add_iv.xml">
```

Comment:
These logicalFilePaths need to match what is already in the dev/alpha/staging/prod databases. These files were also copied into the resources dir so they will be included in the thurloe jar. And the paths need to match between the 2 copies so liquibase does not get confused. This copy (in src/main/migrations) will be removed once jenkins no longer tries to run liquibase.